src/commands/push-to-app-collection.yaml

```
@@ -0,0 +1,41 @@
+parameters:
+  app_name:
+    type: "string"
+  app_catalog:
+    type: "string"
+  app_collection_repo:
+    type: "string"
+steps:
+  - run: |
+        architect project version > version
```

So far "variables" were stored in hidden files like `.version` to avoid conflicts.

```suggestion
        architect project version > .version
```

---

done